community/s6-linux-init/post-install

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+if [ ! -e /etc/s6/current ]; then
+    s6-linux-init-maker -1 -f /etc/s6/skel -G "/usr/bin/agetty -L -8 tty1 115200" -c /etc/s6/current /etc/s6/current
+fi
```

Comment:
What does this do? What if `agetty` is not installed?

Reply:
It generates the ealy booting scripts like loggers, getty, starting service manager, etc.

Should I leave this step to the users instead?

Reply:
That seems like the better option. I don't know much about s6. A Wiki page and short instructions in post-install for s6/etc would be great.

Reply:
Got another idea, it can run s6-linux-init-maker with default options (no early getty, default script location). And if people want to change it later they can. It should be better than guessing what getty people have while keeping the symbolic links pointing to the right files.

See my later commit for what I mean.